### PR TITLE
Add non-normalized confusion matrix option

### PR DESCRIPTION
**Pull request by hannahilea**
Note that it doesn't yet get plotted by default anywhere, the default settings are the same.

### CODE REVIEW

**Review by nbagherzadeh — CHANGES_REQUESTED** [review]
LGTM! Two personal suggestions:

1) What are your thoughts on updating: 

`crange = (0.0, 1.0)` -> ` crange = (0.0, maximum([1.0, maximum(filter(!isnan,confusion))]))` 

so that for the non-normalized outputs, the cmap outputs are not all dark blue. (toy example: if you have a CM = [1, 0; 0, 100], you'd probably want the 100 to be darker than the 1). Adding maximum(1, maximum(args)) will ensure that the normalized figures look similar to previous version. 

test output: 
non-normalized:
image
normalized:
image


2) plot_confusion_matrix! docstring (line 408), sounds like there are only two variants of the confusion matrix; however, we now have a non-normalized option.

**Comment by hannahilea** [comment]
lol yep! great call. 😢 

Screen Shot 2022-06-30 at 4 36 47 PM

**Comment by hannahilea** [comment]
Fixed! Good catch :)
Screen Shot 2022-06-30 at 8 30 25 PM

File: src/plotting.jl [hunk 1]
```diff
@@ -207,7 +210,7 @@ function plot_confusion_matrix!(subfig::FigurePosition, confusion::NumberMatrix,
     ylims!(ax, nclasses + 0.5, 0.5)
     tightlimits!(ax)
     plot_bg_color = to_color(ax.backgroundcolor[])
-    crange = (0.0, 1.0)
+    crange = isnothing(normalize_by) ? (0.0, maximum(filter(!isnan, confusion))) : (0.0, 1.0)
```
Comment (nbagherzadeh) [inline src/plotting.jl:213]: looks good :)

**Review by nbagherzadeh — APPROVED** [review]
LGTM. Plots look great from test 👍 !

**Pull request merged**